package.json

```diff
@@ -149,6 +149,7 @@
     "idb-global": "^2.1.0",
     "iframe-stream": "^3.0.0",
     "inject-css": "^0.1.1",
+    "js-sha3": "^0.8.0",
```

Comment:
Is this really necessary? We already have web3-utils for that right?

Reply:
Yes but i was added after this, we should change this anyway.

Reply:
Deleted now, and the code that was using this, now is using the web3-utils